### DISCUSSION TITLE
Get selected range in plot

### DISCUSSION THREAD

**Discussion by surajdolphy** [danielhrisca/asammdf#619] [Q&A]
Hi,
Currently I am using the python plot function to view my channels. I am working on something which requires me to access the range selected by the user in the plot GUI, i.e., the selected range. Is there a way I can access it once the user moves the cursors?

**Comment by danielhrisca** [comment]
You should connect a handler to the `range_modified` signal https://github.com/danielhrisca/asammdf/blob/2f95d5ca67a83b754eea608b3576b8dbfa15fcf5/asammdf/gui/widgets/plot.py#L2143

**Reply by surajdolphy** [reply]
Hi,
Thank you very much for your prompt response. Unfortunately I am new to Qt. Could you please redirect me to some example how this is done where I can access this signal from the same program from where I call the plot function of asammdf

**Reply by surajdolphy** [reply]
Hi, 
I am sorry I still couldn’t find any heads up on this. 
Please could you guide me on how this can be achieved?

**Reply by JulienGrv** [reply]
asammdf is using [PyQt](https://wiki.python.org/moin/PyQt) for the GUI. So you must start by reading the [PyQt documentation](https://www.riverbankcomputing.com/static/Docs/PyQt5/) if you want to understand how to make modifications. In particular, read this part: https://www.riverbankcomputing.com/static/Docs/PyQt5/signals_slots.html
